Source/Core/Core/IOS/USB/Emulated/Microphone-WiiSpeak.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2025 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+#include "Core/IOS/USB/Emulated/Microphone.h"
+
+namespace IOS::HLE::USB
+{
+class WiiSpeakState;
+
+class MicrophoneWiiSpeak final : public Microphone
+{
+public:
+  MicrophoneWiiSpeak(const WiiSpeakState& sampler);
```

Comment:
```suggestion
  explicit MicrophoneWiiSpeak(const WiiSpeakState& sampler);
```

clang-tidy
>Single-argument constructors must be marked explicit to avoid unintentional implicit conversions